pkg/morph/client/client.go

```diff
@@ -204,6 +206,38 @@ func (c *Client) TestInvoke(contract util.Uint160, method string, args ...any) (
 	return val.Stack, nil
 }
 
+// TestInvokeIterator is the same [Client.TestInvoke] but expands an iterator placed
+// on the stack. Returned values are the values an iterator provides.
+func (c *Client) TestInvokeIterator(contract util.Uint160, method string, args ...any) (res []stackitem.Item, err error) {
+	c.switchLock.RLock()
+	defer c.switchLock.RUnlock()
+
+	if c.inactive {
+		return nil, ErrConnectionLost
+	}
+
+	sid, iter, err := unwrap.SessionIterator(c.rpcActor.Call(contract, method, args...))
+	if err != nil {
+		return nil, fmt.Errorf("iterator expansion: %w", err)
+	}
+
+	items := make([]stackitem.Item, 0)
+	for {
+		ii, err := c.rpcActor.TraverseIterator(sid, &iter, config.DefaultMaxIteratorResultItems)
+		if err != nil {
+			return nil, fmt.Errorf("iterator traversal; session: %s, error: %w", sid, err)
+		}
+
+		if len(ii) == 0 {
+			break
+		}
```

Comment:
How about terminating session?

Reply:
it feels like i have asked about it when sessions were about their release and somebody answers me that it is not critical. but added, no problem

Reply:
> it is not critical

That's true, but if we can release a bit of server resources, then why not perform it.